styles/Join.module.scss

```diff
@@ -3,3 +3,20 @@
   margin: 2rem auto;
   display: block;
 }
+
+button.membershipForm {
+  display: flex;
+  margin: 2rem auto 0;
+  padding: 1rem 2rem;
+  background-color: var(--cyber-dark);
+  border: 3px var(--cyber-gold) solid;
+  border-radius: 1rem;
+  color: var(--cyber-gold);
+  font-family: "Poppins";
+  font-size: larger;
+}
+
+button.membershipForm:hover {
+  background-color: var(--cyber-gold);
+  color: var(--cyber-dark);
```

Comment:
idk if this is a firefox bug but can we add `cursor: pointer;` because that doesn't seem to have applied by default